### PR TITLE
Jump feature

### PR DESCRIPTION
**Pull request by Adwarf14**
Fonction jump du character, définition de la map (soumise à changement) plus les scss associés

### CODE REVIEW

**Review by LeoDiduch — DISMISSED** [review]
That's good

File: frontend/src/components/Map.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.map {
+  position: relative;
+  display: flex;
+  flex-flow: wrap row;
+  width: 832px;
+  height: 320px;
+  border-radius: 40px;
+  margin: 10px auto;
+  background-color: aqua;
+  z-index: 1;
```
Comment (LoicBrassart) [inline frontend/src/components/Map.scss]: Why this `z-index` ?

File: frontend/src/components/Character.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import "./Character.scss";
+
+class Character extends React.Component {
```
Comment (LoicBrassart) [inline frontend/src/components/Character.jsx]: If this component has no internal logic, please use a functional one instead

File: frontend/src/components/Game.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import Map from "./Map";
+
+class Game extends React.Component {
```
Comment (LoicBrassart) [inline frontend/src/components/Game.jsx]: If this component has no internal logic, please use a functional one instead

**Review by LeoDiduch — DISMISSED** [review]
It's all good

File: frontend/src/components/Map.jsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+  componentDidMount() {
+    window.onkeydown = event => {
+      this.getInput(event);
+    };
+  }
+
+  getInput = event => {
+    const keyCode = event.keyCode;
+    if (keyCode === 32) {
```
Comment (LoicBrassart) [inline frontend/src/components/Map.jsx]: Use `key === Space` instead, it's more readable

File: frontend/src/components/Map.jsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+      this.setState({ y: 0.5 });
+      setTimeout(() => {
+        this.setState({ y: 3 });
+      }, 300);
+    }
+  };
+
+  carte = [
```
Comment (LoicBrassart) [inline frontend/src/components/Map.jsx:31]: What purpose have the `0`s ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some things to change

**Review by LoicBrassart — APPROVED** [review]
SnE

**Review by LeoDiduch — APPROVED** [review]

**Pull request merged**